src/RewireState.js

```diff
@@ -126,6 +134,8 @@ export default class RewireState {
 			UNIVERSAL_RESETTER_ID :this.getUniversalResetterID(),
 			UNIVERSAL_WITH_ID :this.getUniversalWithID(),
 			API_OBJECT_ID: this.getAPIObjectID(),
+			INTENTIONAL_UNDEFINED: t.identifier('INTENTIONAL_UNDEFINED'),
+			__INTENTIONAL_UNDEFINED__: '__INTENTIONAL_UNDEFINED__',
```

Comment:
What is this one used for?

Reply:
babel 7's template is more picky about strings that look like placeholders but are not filled in. This was the simplest way to fix it without changing the generated code.